### PR TITLE
fix apiserver crashed when priority classs already exists

### LINKED ISSUE #67367: apiserver crashed when priority classs already exists
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind feature

**What happened**:
When I start the apiserver, found apiserver crased due to the priority class already exists.
```
E0814 06:00:23.516116       1 autoregister_controller.go:190] v1.autoscaling failed with : apiservices.apiregistration.k8s.io "v1.autoscaling" already exists
E0814 06:00:23.516317       1 autoregister_controller.go:190] v1beta1.batch failed with : apiservices.apiregistration.k8s.io "v1beta1.batch" already exists
E0814 06:00:23.516443       1 autoregister_controller.go:190] v2alpha1.batch failed with : apiservices.apiregistration.k8s.io "v2alpha1.batch" already exists
F0814 06:00:23.517195       1 hooks.go:188] PostStartHook "scheduling/bootstrap-system-priority-classes" failed: unable to add default system priority classes: priorityclasses.scheduling.k8s.io "system-cluster-critical" already exists
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`): 1.11.1
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release): ubuntu 16.04
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

when priority classs already exists , ignore error 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #67367 

**Special notes for your reviewer**:

**Release note**:

```release-note
kube-apiserver: fixes error creating system priority classes when starting multiple apiservers simultaneously
```

### CODE REVIEW

File: pkg/registry/scheduling/rest/storage_scheduling.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func AddSystemPriorityClasses() genericapiserver.PostStartHookFunc {
 			return true, nil
 		})
 		// if we're never able to make it through initialization, kill the API server.
-		if err != nil {
+		if err != nil && !apierrors.IsAlreadyExists(err) {
```
Comment (yue9944882) [inline pkg/registry/scheduling/rest/storage_scheduling.go]: i don't really follow but does it make any difference?

Comment (yue9944882) [inline pkg/registry/scheduling/rest/storage_scheduling.go]: [https://github.com/kubernetes/kubernetes/blob/6219e51283a75752f452b7a1fb7c1c3c076bf26f/pkg/registry/scheduling/rest/storage_scheduling.go#L86](https://github.com/kubernetes/kubernetes/blob/6219e51283a75752f452b7a1fb7c1c3c076bf26f/pkg/registry/scheduling/rest/storage_scheduling.go#L86)

i don't really follow but where does the `AlreadyExistsError` come from?

**Comment by liggitt** [comment]
@tanshanshan were you planning on reopening this PR? I think this fix was on the right track

File: pkg/registry/scheduling/rest/storage_scheduling.go (hunk 1 above)
Comment (yue9944882) [reply pkg/registry/scheduling/rest/storage_scheduling.go]: > the AlreadyExists error should be tolerated on line 87, not here

agree. returning a `false, nil` here will give a retry, amiright?

**Pull request merged**